x/pool-incentives/client/cli/cli_test.go

```diff
@@ -203,3 +203,37 @@ func (s *IntegrationTestSuite) TestGetCmdIncentivizedPools() {
 		})
 	}
 }
+
+func (s *IntegrationTestSuite) TestGetCmdExternalIncentiveGauges() {
+	val := s.network.Validators[0]
+
+	testCases := []struct {
+		name      string
+		expectErr bool
+		respType  proto.Message
+	}{
+		{
+			"query external incentivized pools",
+			false, &types.QueryExternalIncentiveGaugesResponse{},
+		},
+	}
```

Comment:
Ditto for CLI command test looks incomplete

Reply:
Same for this comment, I've noticed that the other cli_test.go in any other modules are doing the same thing: only testing if it's getting the right type of response from the query.
Planning to get all these fix at once for all the cli_test.go in one PR if that would be ok!